models/DataObject/Classificationstore.php

```diff
@@ -478,4 +483,21 @@ private function getGroupConfigById(int $groupId): ?Classificationstore\GroupCon
     {
         return Classificationstore\GroupConfig::getById($groupId);
     }
+
+    private function mergeArrays(array $a1, array $a2): array
```

Comment:
LGTM and works, just maybe rename to
```suggestion
    private function mergeArraysByUnion(array $a1, array $a2): array
```
add a phpdoc comment on why this method is needed (to merge without resetting numeric keys or so)

Reply:
nvm on this, he who seeks finds 😄